se3cnn/kernel.py

```diff
@@ -373,13 +375,19 @@ def combination(self, weight):
                     weight_index += mi * mj * b_el
 
                     ker = torch.einsum("uvb,bijxyz->uivjxyz", (w, kij)).contiguous()  # [u, i, v, j, x, y, z]
-                    kernel[si, sj] = ker.view_as(kernel[si, sj])
                 else:
-                    kernel[si, sj] = 0
+                    ker = torch.tensor(0.)
```

Comment:
Please always use things like `weight.new_tensor` to ensure have the same `device` and `dtype`.
Then I think you cannot apply `view` on it. At least `torch.tensor(0.).view(2, 2)` crash on my computer.

Reply:
You could add a test that has at least one `kij is None`.
`se3cnn.kernel.SE3Kernel([(1, 0)], [(1, 0), (1, 5)], 3)` does the trick